aerospike/datadog_checks/aerospike/aerospike.py

```diff
@@ -120,11 +125,29 @@ def check(self, instance):
                 set_tags.extend(namespace_tags)
                 self.collect_info('sets/{}/{}'.format(ns, s), SET_METRIC_TYPE, separator=':', tags=set_tags)
 
+            # https://www.aerospike.com/docs/reference/info/#dcs
+        try:
+            datacenters = self.get_datacenters()
+
+            for dc in datacenters:
+                self.collect_datacenter(dc)
+
+        except Exception as e:
+            self.log.debug("There were no datacenters found: %s" % e)
```

Comment:
What exactly causes an error? `get_datacenters`?

Reply:
Yeah it fails when there are no datacenters to be collected since that's an enterprise feature. 

https://www.aerospike.com/docs/reference/info/#dcs